instaloader/structures.py

```diff
@@ -1073,6 +1085,13 @@ def owner_id(self) -> int:
         """The ID of the StoryItem owner."""
         return self.owner_profile.userid
 
+    @property
+    def get_mentions(self) -> List[dict]:
+        """get the user mentions in the story"""
+        tappable_objects = self._node['tappable_objects']
+        mentions = [obj for obj in tappable_objects if obj['__typename'] == 'GraphTappableMention']
+        return mentions
+
```

Comment:
- Property names should not start with "get"
- It would be much nicer to get `Profile` instances instead of unspecific dicts
- """Mentioned :class:`Profile`s in the StoryItem""" might then be a better docstring